reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -169,9 +173,12 @@ public static Scheduler parallel() {
 	}
 
 	/**
-	 * Executes tasks on the caller's thread immediately.
+	 * Executes tasks immediately, on the thread that submitted it (eg. the
+	 * thread on which an operator is currently processing its onNext/onComplete/onError signals).
+	 * This {@link Scheduler} is typically used as a "null object" for APIs that require a
+	 * Scheduler but one doesn't want to change threads.
 	 *
-	 * @return a reusable {@link Scheduler}
+	 * @return a reusable {@link Scheduler} that schedules on the current thread
```

Comment:
nit: could use a bit of rephrasing to avoid the thread reference too (e.g. "that instantly runs the submitted Runnable")

Reply:
done